services/core-web/src/components/mine/Permit/MinePermitTable.tsx

```diff
@@ -484,71 +255,177 @@ export const MinePermitTable: React.FC<MinePermitTableProps> = ({
-              <Dropdown className="full-height full-mobile" overlay={menu} placement="bottomLeft">
-                {/* @ts-ignore */}
-                <Button type="secondary" className="permit-table-button">
-                  <div className="padding-sm">
-                    <img className="padding-sm--right icon-svg-filter" src={EDIT} alt="Add/Edit" />
-                    Add/Edit
-                    <img
-                      className="padding-sm--right icon-svg-filter"
-                      src={CARAT}
-                      alt="Menu"
-                      style={{ paddingLeft: "5px" }}
-                    />
-                  </div>
-                </Button>
-              </Dropdown>
-            </AuthorizationWrapper>
-            <AuthorizationWrapper permission={Permission.ADMIN}>
-              {deletePermitPopUp}
-            </AuthorizationWrapper>
-          </div>
-        );
+        const title =
+          issues.length > 0
+            ? "You cannot delete this permit due to the following issues:"
+            : "Are you sure you want to delete this permit and all related permit amendments and permit documents?";
+        const content =
+          issues.length > 0 ? (
+            <div style={{ whiteSpace: "pre-wrap" }}>
+              <ul>
+                {issues.map((issue) => (
+                  <li key={issue}>{issue}</li>
+                ))}
+              </ul>
+            </div>
+          ) : null;
+        const onOk = isDeletionAllowed
+          ? () => handleDeletePermit((record.permit as IPermit).permit_guid)
+          : () => {};
+        return Modal.confirm({
+          title,
+          content,
+          onOk,
+          okText: isDeletionAllowed ? "Delete" : "Ok",
+          cancelText: "Cancel",
+        });
       },
     },
-  ];
+  ].filter(Boolean);
+
+  const recordActionsFilter = (record, actionItems) => {
+    const filterOut = record.hasAmalgamated
+      ? ["amalgamate", "add-amendment"]
+      : ["amalgamate-amend"];
+    return actionItems.filter(({ key }) => !filterOut.includes(key));
+  };
+
+  const actionsMenu = renderActionsColumn({ actions, recordActionsFilter });
+
+  const columns: ColumnsType<MinePermitTableItem> = [
+    {
+      title: "Permit No.",
+      key: "permitNo",
+      render: (record) => <div title="Permit No.">{renderPermitNo(record.permit)}</div>,
+    },
+    renderCategoryColumn("status", "Status", permitStatusOptionsHash),
+    renderTextColumn("permittee", "Permittee"),
+    renderTextColumn("firstIssued", "First Issued"),
+    renderTextColumn("lastAmended", "Last Amended"),
+    userCanEditPermits && actionsMenu,
+  ].filter(Boolean);
+
+  const handleNavigateToPermitConditions = (record) => {
+    return history.push(
+      route.EDIT_PERMIT_CONDITIONS.dynamicRoute(record.mineGuid, record.permit_amendment_guid)
+    );
+  };
+
+  // from DownloadAllDocumentsButton/MineReportTable.js - I think there is a new way to accomplish this
```

Comment:
Want to put migrating this over to use the new zip file download functionality ( I can't find one in the backlog)?
Think this must be one of the last bulk download that doesn't